src/main/resources/hibernate.cfg.xml

```diff
@@ -4,11 +4,11 @@
         "http://www.hibernate.org/dtd/hibernate-configuration-3.0.dtd">
 <hibernate-configuration>
     <session-factory>
-        <property name="connection.url">${database.connection.url}</property>
+        <property name="connection.url">jdbc:mysql://localhost:3306/covid_pulse</property>
         <property name="connection.driver_class">com.mysql.cj.jdbc.Driver</property>
 
-        <property name="connection.username">${database.user}</property>
-        <property name="connection.password">${database.password}</property>
+        <property name="connection.username">devfecta</property>
```

Comment:
I know these profiles weren't working right for you, but we should get rid of these hard coded values before merging into master.